src/Users/User.php

```diff
@@ -276,4 +294,49 @@ public function setInitials($initials): User
 
         return $this;
     }
+
+    public function getJobTitle(): ?string
+    {
+        return $this->jobTitle;
+    }
+
+    /**
+     * @param string|null $jobTitle
+     */
+    public function setJobTitle($jobTitle): User
```

Comment:
Since we're targeting PHP 7.3 now, we could probably take all the setters and remove the phpdoc and replace the params with `?string` or `?int`, etc.  This isn't something to hold up this PR, just thinking we should probably include that in v3.

Reply:
I love it, added this commit in 👍🏾